br-webui/index.js

```diff
@@ -734,10 +734,34 @@ io.on('connection', function(socket) {
 		cmd.on('error', (err) => {
 			logger.log('Failed to start child process.');
 			logger.log(err.toString());
-		});	
+		});
 	});
-	
-	
+
+	socket.on('save params', function(data) {
+		var file_path = _companion_directory + data.file
```

Comment:
We should save the persistent configuration outside of the repo. I think we should change this to ~ + data.file